Snakefile

```diff
@@ -15,9 +13,12 @@ include: "rules/potential.smk"
 include: "rules/sync.smk"
 
 localrules: all, clean
+root_dir = config["root-directory"] + "/" if config["root-directory"] not in ["", "."] else ""
+__version__ = open(f"{root_dir}VERSION").readlines()[0].strip()
+script_dir = f"{root_dir}scripts/"
```

Comment:
Hmm, I'm not sure I see that. Yes, the individual python files have a bunch of cross-dependencies, but it isn't formed as a package otherwise. Irrespective of whether the contents of `scripts` (formerly `src`) is a collection of scripts of a package, the snakemake syntax is to call `script:`. Given that all the files we point to when making that call are in one directory, it then makes sense to call that directory `scripts`, no?